mantidproject/mantid
# H5Utils

This is a central location to collect common usage of the [HDF high level API](https://www.hdfgroup.org/HDF5/doc/cpplus_RM/index.html). It is not intended to be complete.

**To test:**

Should be a code review. Make new issues for missing functionality. However, errors need to be fixed.

_Does not need to be in the release notes._ It is a developer facing change.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
